packages/odata-service-writer/templates/extend/manifest.json

```diff
@@ -6,15 +6,15 @@
                 "type": "OData",
                 "settings": {
                     "annotations": [<%if (locals.annotations && annotations.technicalName) {%>
-                        "<%- annotations.technicalName %>"<% if (locals.localAnnotationsName) {%>,<%}%><% } %><% if (locals.localAnnotationsName) { %>
+                        "<%- annotations.name %>"<% if (locals.localAnnotationsName) {%>,<%}%><% } %><% if (locals.localAnnotationsName) { %>
```

Comment:
Would safer to change if to check `name` instead of `technicalName` as `name` is now referenced in the template. 
`"annotations": [<%if (locals.annotations && annotations.name) {%>`
Anyway from the code it looks like it always populated based on `setDefaultAnnotationsName`

Reply:
Thanks, I may change it in a future pr